### PR TITLE
Fix commandline flags

### PR DESCRIPTION
**Pull request by phil9909**

### CODE REVIEW

File: istio-control/istio-config/templates/deployment.yaml [hunk 1]
```diff
@@ -36,9 +36,7 @@ spec:
           - containerPort: 15019
           command:
           - /usr/local/bin/galley
-{{- if and (.Values.global.istioVersion) (semverCompare "<1.2" .Values.global.istioVersion) }}
```
[the remark this answers is not in the document]
Comment (phil9909) [reply istio-control/istio-config/templates/deployment.yaml]: Yes, this was cherry-picked from 1.1 into master https://github.com/istio/istio/commit/f7feb0fcbab73d05375815f06956ac3002f231d6

[the remark this answers is not in the document]
Comment (loewenstein) [reply istio-control/istio-config/templates/deployment.yaml]: So for supporting 1.0 the switch would still be necessary?

@costinm should the installer strive for 1.0 support?

[the remark this answers is not in the document]
Comment (costinm) [reply istio-control/istio-config/templates/deployment.yaml]: I don't think we need to at this point. It did because it was started around 0.8. 

We do need to test that the installer can work alongside 1.0, i.e. if someone has 1.0 installed in istio-system they can deploy 1.1 or 1.2 using the new installer.

File: security/citadel/templates/deployment.yaml [hunk 2]
```diff
@@ -35,7 +35,11 @@ spec:
           args:
             - --append-dns-names=true
             - --grpc-port=8060
+          {{- if and (.Values.global.istioVersion) (semverCompare "<1.2" .Values.global.istioVersion) }}
```
Comment (costinm) [inline security/citadel/templates/deployment.yaml]: Can we revert the change in citadel instead ( or make sure they maintain proper backward compat between releases ?)
